nearit-ui/src/main/res/layout/nearit_ui_layout_permission_snackbar_button.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The button layout is a bit different from the mock, it doesn't seem large enough and the test should be bold

Reply:
done